public/js/home/reply-product.js

```diff
@@ -17,7 +18,7 @@ $(document).ready(function ($) {
                     '<input type="hidden" name="product_id" value="' + productId + '">' +
                     '<input type="hidden" name="parent_id" value="' + parentId + '">' +
                 '</form>';
-
+        replyForm.find('.form-comment').remove();
         replyForm.append(form);
         var textarea = replyForm.find('.input-comment');
         $( textarea , this ).focus();
```

Comment:
$( textarea , this ).focus(); cách linh tinh thế

Reply:
để n focus vào cái input đấy